platinum_searcher.go

```diff
@@ -32,7 +32,7 @@ func (p PlatinumSearcher) Run(args []string) int {
 	conflag.LongHyphen = true
 	conflag.BoolValue = false
 	for _, c := range []string{
-		filepath.Join(os.Getenv("XDG_CONFIG_HOME"), "pt", "config.toml"),
+		filepath.Join(xdgConfigHomeDir(), "pt", "config.toml"),
 		filepath.Join(home.Dir(), ".ptconfig.toml"),
```

Comment:
That would break compatibility for users who are currently using `$HOME/.ptconfig.toml` so I'm not sure that's a good idea.